doc/examples/applications/plot_tool_using_holoviz.py

```diff
@@ -0,0 +1,189 @@
+"""
+=======================================
+Interactive Tools and Apps with HoloViz

```

Comment:
```suggestion
Explore and annotate image data interactively with with HoloViz
```
Please write titles at the imperative and without capitalizing non-first words. This convention also applies to section titles. Thanks!

PS: How do 'HoloViz' and 'HoloViews' relate?

Reply:
HoloViz provides a set of packages. HoloViews is one such package. Panel is another such package. See https://holoviz.org/